package.json

```diff
@@ -89,5 +90,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "optionalPeerDependencies": {
```

Comment:
Is this a real thing? I don't see it documented anywhere, except for this library: https://github.com/Wizcorp/codependency